### DISCUSSION TITLE
525.47.34 Vulkan driver still not released?

### DISCUSSION THREAD

**Discussion by oscarbg** [NVIDIA/open-gpu-kernel-modules#536] [Q&A]
Has been pulled?

**Comment by visious25** [comment]
Will they add vulkan driver in the future?

**Comment by ghost** [comment]
Am I correct, is Vulkan not supporting in nvidia open drivers 535.104.05 or have I misconfigured something?

The problem is that I didn't find any explicit mention of Vulkan support or lack thereof in the documentation and thought it was something I hadn't configured.

NVIDIA GeForce RTX 3060 Laptop GPU 0x2520.
openSUSE Tumbleweed.
